README.md

```diff
@@ -68,43 +67,12 @@ system as background processes. It provides functionality to:
 
 Requirements:
 
-1.  Python 2.7, 3.6 and higher
+1.  Python 3.8 and higher
```

Comment:
Probably not necessary since it is really all python versions allowed for building.

Reply:
I think a reference to "all python versions allowed for building" would not be very helpful for users who just want to install the package.

The question is therefore if we don't provide an easy to read statement like the above, where do we point to instead?
 
Note that not everybody installs from Pypi. For example, the package could be made part of a Linux distro (like for pywbem).

I think the price to change this version in the docs is very small, so I would just leave it as it is.

Reply:
We discussed and concluded to leave the statement in.